packages/docs/components/button.md

```diff
@@ -486,30 +490,30 @@ Color is not a clear affordance for all users, please use clear, concise copy to
   ```
 </figure>
 
-## Overlay
+## Dismiss
 
 <figure class="docs-example">
   <div class="docs-example--rendered is-rendered-success">
-    <button class="ods-button is-ods-button-overlay">
-      <span class="ods-button--label">Overlay</span>
+    <button class="ods-button is-ods-button-dismiss">
+      <span class="ods-button--label">Dismiss</span>
     </button>
-    <button class="ods-button is-ods-button-overlay is-ods-button-hover">
+    <button class="ods-button is-ods-button-dismiss is-ods-button-hover">
       <span class="ods-button--label">Hover</span>
     </button>
-    <button class="ods-button is-ods-button-overlay is-ods-button-focus">
+    <button class="ods-button is-ods-button-dismiss is-ods-button-focus">
       <span class="ods-button--label">Focus</span>
     </button>
-    <button class="ods-button is-ods-button-overlay" disabled>
+    <button class="ods-button is-ods-button-dismiss" disabled>
       <span class="ods-button--label">Disabled</span>
     </button>
   </div>
```

Comment:
Still hesitant on including this as a button mode. Should this be exposed publicly? I would suggest keeping it private until we establish a need.

Reply:
Maybe im overthinking this...

Reply:
@joejezowski-okta are you including this in the Figma kit? If not, I'm happy to remove from the docs for now.

Alternately, we could ship as is and re-evaluate whether this is Private or Public prior to 1.0.

Items like this feel central to the "what do we provide for snowflakes" question.